source/Conversation.cpp

```diff
@@ -271,17 +270,35 @@ void Conversation::Save(DataWriter &out) const
 			}
 			if(node.isChoice)
 			{
-				out.Write(node.data.empty() ? "name" : "choice");
+				out.Write(node.elements.empty() ? "name" : "choice");
 				out.BeginChild();
 			}
-			for(const auto &it : node.data)
+			for(const auto &it : node.elements)
 			{
 				// Break the text up into paragraphs.
-				for(const string &line : Format::Split(it.first, "\n"))
+				for(const string &line : Format::Split(it.text, "\n"))
+				{
 					out.Write(line);
+					// If the conditions are the same, output them for each
+					// paragraph. (We currently don't merge paragraphs with
+					// identical ConditionSets, but some day we might.
+					if(!it.conditions.IsEmpty())
+					{
+						out.BeginChild();
+						{
+							out.Write("to", "display");
```

Comment:
```suggestion
							out.Write("to display");
```
easier and a tiny bit better performance

Reply:
Doing this means needing to change the parsing code further down. That expects `to` and `display` to each be their own token, with this change, it would become `"to display"`, a single token. I'm not entirely sure it should be a single token. While we may not have other things after the `to` now, we might in the future, and other places where something similar is present (missions, for example), two separate tokens are used.

Reply:
If you want it like this I am ok with it